test/unit/math/opencl/prim/add_test.cpp

```diff
@@ -188,6 +203,53 @@ TEST(MathMatrixCL, add_value_check) {
   EXPECT_EQ(9, m3(2, 0));
   EXPECT_EQ(12, m3(2, 1));
   EXPECT_EQ(17, m3(2, 2));
+
+  matrix_cl<double> m11s = add(m11, 1.5);
+  m3 = stan::math::from_matrix_cl(m11s);
+  EXPECT_EQ(2.5, m3(0, 0));
+  EXPECT_EQ(3.5, m3(0, 1));
+  EXPECT_EQ(4.5, m3(0, 2));
+  EXPECT_EQ(5.5, m3(1, 0));
+  EXPECT_EQ(6.5, m3(1, 1));
+  EXPECT_EQ(7.5, m3(1, 2));
+  EXPECT_EQ(8.5, m3(2, 0));
+  EXPECT_EQ(9.5, m3(2, 1));
+  EXPECT_EQ(10.5, m3(2, 2));
```

Comment:
I think you could use `EXPECT_MATRIX_NEAR` macro that is used in many tests and some Eigen code to simplify these tests.